templates/uefi/generic-qemu-uefi-boot-template.jinja2

```diff
@@ -0,0 +1,62 @@
+{% extends 'base/kernel-ci-base.jinja2' %}
+{% set ctx_arch = arch %}
+{% if arch == "arm" %}
+{% set ctx_arch = 'arm64' %}
+{% set ctx_cpu = 'cortex-a15' %}
+{% set console_dev = 'ttyAMA0' %}
+{% endif %}
+{% if arch == "arm64" %}
+{% set ctx_arch = 'arm64' %}
+{% set console_dev = 'ttyAMA0' %}
+{% endif %}
+{% if arch == 'x86_64' %}
+{% set ctx_arch = 'amd64' %}
+{% set console_dev = 'ttyS0' %}
+{% endif %}
+{% if arch == 'i386' %}
+{% set ctx_arch = 'i386' %}
+{% set ctx_cpu = 'host' %}
+{% set console_dev = 'ttyS0' %}
+{% endif %}
+{% block metadata %}
+{{ super() }}
+{% endblock %}
+{% block main %}
+{{ super() }}
+{% endblock %}
+{% block actions %}
+context:
+  arch: {{ ctx_arch }}
+  cpu: {{ ctx_cpu }}
+  guestfs_interface: virtio
+{% if arch == 'arm' or arch == 'arm64' %}
+  machine: virt,{{ ctx_gicversion }}
+{% endif %}
+
+actions:
+- deploy:
+    timeout:
+      minutes: 3
+    to: tmpfs
+    os: oe
+    images:
+      bios:
```

Comment:
Same thing here, this could be solved in a generic way in the base template:
```
{% if bios_url %}
    bios:
      image_arg: '-bios {bios}'
      url: {{ bios_url }}
{% endif }
```